packages/menu/menu-item.md

```diff
@@ -78,41 +104,40 @@ Content assigned to the `description` slot will be placed below the `<sp-menu-it
 </sp-menu>
 ```
 
-### Value slot
+#### Value
 
-Content assigned to the `value` slot will be placed at the end of the `<sp-menu-item>`, like values, keyboard shortcuts, etc., based on the current text direction.
-
-```html
-<sp-menu style="width: 200px;">
-    <sp-menu-item>Save<kbd slot="value">⌘S</kbd></sp-menu-item>
-    <sp-menu-item>Completed<span slot="value">47%</apn></sp-menu-item>
-    <sp-menu-item>Activity<sp-link slot="value" href="#">More&nbsp;info</sp-link></sp-menu-item>
-</sp-menu>
-```
-
-### Submenu
+##### Value slot
 
-An `<sp-menu-item>` can also accept content addressed to its `submenu` slot. Using the `<sp-menu>` element with this slot name the options will be surfaced in flyout menu that can be activated by hovering over the root menu item with your pointer or focusing the menu item and pressing the appropriate `ArrowRight` or `ArrowLeft` key based on text direction to move into the submenu.
+Content assigned to the `value` slot will be placed at the end of the `<sp-menu-item>`, like values, keyboard shortcuts, etc., based on the current text direction.
 
 ```html
 <sp-menu style="width: 200px;">
     <sp-menu-item>
-        Item with submenu
-        <sp-menu slot="submenu">
-            <sp-menu-item>Additional options</sp-menu-item>
-            <sp-menu-item>Available on request</sp-menu-item>
-        </sp-menu>
+        Save
+        <kbd slot="value">⌘S</kbd>
+    </sp-menu-item>
+    <sp-menu-item>
+        Completed
+        <span slot="value">47%</span>
+    </sp-menu-item>
+    <sp-menu-item>
+        Activity
+        <sp-link slot="value" href="#">More&nbsp;info</sp-link>
     </sp-menu-item>
 </sp-menu>
 ```
 
-### Value attribute
+#### Value attribute
 
 When displayed as a descendent of an element that manages selection (e.g. `<sp-action-menu>`, `<sp-picker>`, `<sp-split-button>`, etc.), an `<sp-menu-item>` will represent the "selected" value of that ancestor when its `value` attribute or the trimmed `textContent` (represeted by `el.itemText`) matches the `value` of the ancestor element.
 
 In the following example, the selected `<sp-menu-item>` represents a `value` of `Text that is really long and useful to a visitor, but not exactly good to use in your application or component state.` for the ancestor element.
```

Comment:
```suggestion
In the following example, the selected `<sp-menu-item>` represents a `value` of `"Text that is really long and useful to a visitor, but not exactly good to use in your application or component state."` for the ancestor element.
```
We could probably remove the backticks around the `value`'s value if you're keen.